dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/trackedentity/TrackedEntityParamsMapper.java

```diff
@@ -76,14 +76,14 @@
 
 /**
  * Maps query parameters from {@link TrackedEntitiesExportController} stored in
- * {@link TrackedEntityCriteria} to {@link TrackedEntityQueryParams} which is
- * used to fetch tracked entities from the DB.
+ * {@link RequestParams} to {@link TrackedEntityQueryParams} which is used to
+ * fetch tracked entities from the DB.
  *
  * @author Luciano Fiandesio
  */
-@Component( "org.hisp.dhis.webapi.controller.tracker.export.trackedentity.EntityCriteriaMapper" )
+@Component( "org.hisp.dhis.webapi.controller.tracker.export.trackedentity.EntityParamsMapper" )
```

Comment:
I don't think we need the bean id if we are not implementing an interface and there is not some sort of conflict https://github.com/dhis2/wow-backend/blob/master/guides/spring_conventions.md#bean-id